src/Sulu/Bundle/AdminBundle/Resources/js/views/List/List.js

```diff
@@ -106,12 +107,18 @@ class List extends React.Component<Props> {
             props.resourceStore
         );
 
+        const metadataOptions = this.buildMetadataOptions(
+            attributes,
+            routerAttributesToListMetadata
+        );
+
         this.listStore = new ListStore(
             resourceKey,
             listKey,
             userSettingsKey,
             observableOptions,
-            listStoreOptions
+            listStoreOptions,
+            metadataOptions
```

Comment:
Is it possible that the passing of these options is not tested? Would be important IMO, because these options are not used in the core itself. That might result in breaking change for the form bundle without us realizing it...